src/app/content/utils.ts

```diff
@@ -20,37 +21,35 @@ export { stripIdVersion } from './utils/idUtils';
 export { scrollSidebarSectionIntoView } from './utils/domUtils';
 
 export interface ContentPageRefencesType {
-  bookId?: string;
-  bookVersion?: string;
+  bookId: string;
+  bookVersion: string;
```

Comment:
book version is still optional in this structure because it is not in the cross-book links